modules/ipi-install-configure-network-components-to-run-on-the-control-plane.adoc

```diff
@@ -4,9 +4,12 @@
 
 :_content-type: PROCEDURE
 [id='configure-network-components-to-run-on-the-control-plane_{context}']
-= Optional: Configuring network components to run on the control plane
+= Configuring network components to run on the control plane
 
+[IMPORTANT]
```

Comment:
A module needs to start with a paragraph, not a note. If you need to add the note, you must also add a separate introductory statement that describes the scope of the module.